engine/graphics/src/opengl/graphics_opengl.cpp

```diff
@@ -72,7 +72,7 @@
 #endif
```

Comment:
Since we are talking about a target that may want to see both OpenGL and OpenGLES, I think there needs to be some design decisions:
* Should we only support OpenGLES?
* How to select between the targets? Compile time or runtime?

We could for instance easily compile two versions of this file (graphics_opengl.cpp` into two libraries `libgraphics_opengl.a` and `libgraphics_opengles.a`, only difference is using different defines (USE_OPENGLES).

Or, do we want to be able to check at runtime?
(I'm not even sure it's even possible, given that they use different headers+libraries)
And it also requires the built project to include two versions of the same shaders, which might not be what one wants.

My gut feeling is that the two-library-approach is the easier route to take.

Reply:
I think you can technically link both desktop and ES into the same binary and then load the correct symbols depending on which of the libraries you use. I think there's loaders like GLAD or GLEW that we could use instead of our current setup, but I'd like to take the shortest path for this as possible. And that likely means building two libraries that use different headers and libraries.